skyportal/models/gcn.py

```diff
@@ -274,14 +274,12 @@ def get_html(self):
             data = json.loads(data)
 
         # Create the filters mapper
-        if data.get("sources"):
-            from skyportal.handlers.api.photometry import get_bandpasses_to_colors
-
+        if "sources" in data and data["sources"] is not None:
```

Comment:
any reason you don't want to use `data.get("sources"):` anymore?

Reply:
No reason at all.